tracking_manager/models/ir_model.py

```diff
@@ -0,0 +1,205 @@
+# Copyright (C) 2022 Akretion (<http://www.akretion.com>).
+# @author Kévin Roche <kevin.roche@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from ast import literal_eval
+
+from odoo import api, fields, models, tools
+
+
+class IrModel(models.Model):
+    _inherit = "ir.model"
+
+    active_custom_tracking = fields.Boolean(default=False)
```

Comment:
```suggestion
    active_custom_tracking = fields.Boolean()
```
no need of default `False` value on Boolean